packages/eslint-config-ckeditor5/.eslintrc.js

```diff
@@ -315,12 +315,156 @@ module.exports = {
 		'mocha/no-sibling-hooks': 'error',
 		'mocha/no-top-level-hooks': 'error'
 	},
-	'overrides': [
+	overrides: [
 		{
-			'files': [ '**/tests/**/*.js' ],
-			'rules': {
+			files: [ '**/*.ts' ],
+			plugins: [
+				'@typescript-eslint',
+				'ckeditor5-rules',
+				'mocha'
+			],
+			parser: '@typescript-eslint/parser',
+			extends: [
+				'eslint:recommended',
+				'plugin:@typescript-eslint/eslint-recommended',
+				'plugin:@typescript-eslint/recommended'
+			],
+			rules: {
+				'@typescript-eslint/ban-types': [
+					'error',
+					{
+						types: { Function: false },
```

Comment:
When expecting a callback, should we define a custom definition instead of the `Function` type? For instance:

```ts
interface FooCallback {
  (): string
}
```

Reply:
Yes, that's true. But in some places, we treat functions in a dynamically-typed language way. Like in `EventEmitter`. Using Function there is still better than `any`.